lib/route_writer.dart

```diff
@@ -0,0 +1,60 @@
+import 'dart:convert';
+import 'dart:io';
+import 'dart:math';
+
+import 'package:sail_routing_dart/cart_point.dart';
+import 'package:sail_routing_dart/route_model.dart';
+
+class RouteWriter {
+  writeToFile({List<double> start, List<double> end, List<List<double>> points, double wind_direction}) {
+    List fileContents = new List();
+    fileContents.add({'start': start});
+
+    fileContents.add({'end': end});
+    fileContents.add({'wind_direction': wind_direction});
+
+    fileContents.add({'points': points});
+
+    var myFile = new File('lib/plotter_data.json');
+
+    var sink = myFile.openWrite();
+    sink.write(json.encode(fileContents));
+    sink.close();
+  }
+
+  writeToFileFromRouteModel({RouteModel route}) {
+    List fileContents = new List();
+    fileContents.add({'start': route.start.toList()});
+
+    fileContents.add({'end': route.end.toList()});
+    fileContents.add({'wind_direction': route.wind_radians});
+
+    fileContents.add({'points': route.intermediate_points.map((CartPoint p) => p.toList()).toList()});
+
+    var myFile = new File('lib/plotter_data.json');
```

Comment:
This is going to be a problem running dart other than from main, can we look into using some sort of package path?

Reply:
Or be able to pass the file path in as an arg?